aws-lambda/pom.xml

```diff
@@ -21,13 +21,13 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.camel.quarkus.examples</groupId>
     <artifactId>camel-quarkus-examples-aws-lambda</artifactId>
-    <version>2.4.0</version>
+    <version>2.5.0-SNAPSHOT</version>
```

Comment:
This should either be 2.5.0 or you maybe want to make as separate commit to tag 2.5.0 of the examples?

Reply:
Yes that was you who wrote the doc 🤣
I was going to suggest changing that part. Let me update this and also the release guide.

Reply:
Indeed, the tag commit comes after the PR in the docs. Sorry for the noise.